apps/emqx/src/emqx_tls_lib.erl

```diff
@@ -309,11 +309,18 @@ trim_space(Bin) ->
 %% `keyfile', `certfile' and `cacertfile' can be either pem format key or certificates,
 %% or file path.
 %% When PEM format key or certificate is given, it tries to to save them in the given
-%% sub-dir in emqx's data_dir, and replace saved file paths for SSL options.
+%% sub-dir in emqx's certs data_dir, and replace saved file paths for SSL options.
+%% i.e. `data/certs/<GIVEN_SUB_DIR>'
+%% Set the `in_mutable_dir' option to `false' to store the cert file directly
+%% in the given dir instead of in the mutable directory
```

Comment:
rename the old to `ensure_ssl_files_in_data_dir/2`.
and add a new `ensure_ssl_files/2` to respect the input dir as it is.
 
`ensure_ssl_files_in_data_dir/2` should call `ensure_ssl_files/2`.